### PR TITLE
Set as background flyout

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Related #7834

**Details of Changes**
Add details of changes here.
- Added a flyout to the set as "background" button that shows desktop and lockscreen background. 

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Should the text say set as background?

**Comment by puppetsw** [comment]
> Should the text say set as background?

Yep, it will once the split button is styled.

File: src/Files.Uwp/ViewModels/ToolbarViewModel.cs [hunk 1]
```diff
@@ -1226,6 +1228,7 @@ public List<ListedItem> SelectedItems
                     OnPropertyChanged(nameof(IsInfFile));
                     OnPropertyChanged(nameof(IsPowerShellScript));
                     OnPropertyChanged(nameof(IsImage));
+                    OnPropertyChanged(nameof(IsMultipleImages));
```
Comment (yaichenbaum) [inline src/Files.Uwp/ViewModels/ToolbarViewModel.cs:1231]:
```suggestion
                    OnPropertyChanged(nameof(IsMultipleImagesSelected));
```

File: src/Files.Uwp/Views/ModernShellPage.xaml.cs [hunk 2]
```diff
@@ -254,6 +254,7 @@ private void InitToolbarCommands()
             ToolbarViewModel.RunWithPowerShellCommand = new RelayCommand(async () => await Win32Helpers.InvokeWin32ComponentAsync("powershell", this, PathNormalization.NormalizePath(SlimContentPage?.SelectedItem.ItemPath)));
             ToolbarViewModel.PropertiesCommand = new RelayCommand(() => SlimContentPage?.CommandsViewModel.ShowPropertiesCommand.Execute(null));
             ToolbarViewModel.SetAsBackgroundCommand = new RelayCommand(() => SlimContentPage?.CommandsViewModel.SetAsDesktopBackgroundItemCommand.Execute(null));
+            ToolbarViewModel.SetAsLockScreenCommand = new RelayCommand(() => SlimContentPage?.CommandsViewModel.SetAsLockscreenBackgroundItemCommand.Execute(null));
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/ModernShellPage.xaml.cs:257]:
```suggestion
            ToolbarViewModel.SetAsLockScreenBacgrkoundCommand = new RelayCommand(() => SlimContentPage?.CommandsViewModel.SetAsLockscreenBackgroundItemCommand.Execute(null));
```

File: src/Files.Uwp/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -2795,4 +2795,10 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ReviewFilesContent" xml:space="preserve">
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen background</value>
+  </data>
+  <data name="SetAsText" xml:space="preserve">
+    <value>Set as desktop background</value>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/en-US/Resources.resw:2802]: Are there existing strings for this?

Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2802]: Yeah, there probably are. I'll take a look.

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2802]: What about "Set as wallpaper" with the sub options being "Desktop", "Lockscreen", and "Both"?

Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2802]: That sounds like a good idea. I'll make a separate PR for the context menu.

**Comment by puppetsw** [comment]
When multiple images are selected the `Set as desktop background` changes to `Set as desktop slideshow`

**Comment by yaichenbaum** [comment]
image

**Comment by yaichenbaum** [comment]
image

Can the main button say "Set as background" and the sub options be "Desktop" and "Lockscreen"?

When multiple images are selected, I'd add a separate button without the flyout that says "Set as desktop slideshow".

**Comment by puppetsw** [comment]
> image

I think that happened when I merged yours in.

**Comment by yaichenbaum** [comment]
I recommend doing a rebase next time

**Comment by puppetsw** [comment]
> I recommend doing a rebase next time since it should avoid issues like.

Will do.

**Pull request closed without merging**